app/src/ingestion/elements.py

```diff
@@ -0,0 +1,34 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import List
+
+
+class TextType(Enum):
+    NARRATIVE_TEXT = "NarrativeText"
+    LIST_ITEM = "ListItem"
+    LIST = "List"
+
+
+@dataclass
+class Page:
+    # The actual page in the document
+    pdf_page_number: int
+
+    # The label of the page in the document
+    # E.g., "i", "ii", "1", "2", etc.
+    document_page_number: str
+
+
+@dataclass
+class Heading:
+    title: str
+    level: int
+    page: Page
+
+
+@dataclass
+class EnrichedText:
```

Comment:
It felt a bit like overdoing it to add a new class `GroupedText` with the exact same fields, but open to the team's thoughts here!

Reply:
We don't need to create another class. The tech spec uses different names to distinguish the differences of the I/O for each step to avoid confusion.